apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -184,7 +180,15 @@ function wpcom_track_global_styles( $blog_id, $post, $updated ) {
  * @return bool Returns true if custom styles are in use.
  */
 function wpcom_global_styles_in_use() {
-	$user_cpt = WP_Theme_JSON_Resolver_Gutenberg::get_user_data_from_wp_global_styles( wp_get_theme() );
+	if ( class_exists( 'WP_Theme_JSON_Resolver' ) ) {
```

Comment:
Can we invert the condition and early return? This way, the IDE won't scream at us that $user_cpt might not be defined (even though that's not possible).

This way, we won't need the `else` and $user_cpt is defined outside of a condition (and easier to mentally parse).

Reply:
Good suggestion! I'll update it after our support rotation.